src/app/AttributeCache.h

```diff
@@ -217,6 +218,8 @@ class AttributeCache : protected ReadClient::Callback
      */
     CHIP_ERROR Get(const ConcreteAttributePath & path, TLV::TLVReader & reader);
 
+    CHIP_ERROR GetVersion(EndpointId mEndpointId, ClusterId mClusterId, Optional<DataVersion> & aVersion);
```

Comment:
Documentation as to when optional will have value vs. not. And when you'll get errors returne dback.

Reply:
updated, thanks